core/src/main/groovy/noe/common/utils/Cmd.groovy

```diff
@@ -854,7 +854,7 @@ public class Cmd {
         try {
           javaPids.add(Integer.parseInt(pid))
         } catch (NumberFormatException ex) {
-          throw new NumberFormatException("Guys, I can't parse Integer from:${pid}", ex)
```

Comment:
Is there a reason for this change?

Reply:
There is no constructor NumberFormatException(String, Throwable) which means that this sentence was giving an obscure groovy error masking the $pid and the original Exception.

On further thought, I think the code would be cleaner as something like

```
} catch (NumberFormatException ex) {
   log.error("Error parsing Integer from pid: ${pid}")
   throw ex
}
```
So as not to loose information in the stack trace which still happens.